shared/fs/filepreview/normal-preview.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import * as I from 'immutable'
+import * as React from 'react'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+import {globalStyles, globalColors, globalMargins, platformStyles} from '../../styles'
+import {Box} from '../../common-adapters'
+import Footer from '../footer/container'
+import Header from './header-container'
+import {getDisplayComponent} from './common'
+
+type NormalPreviewProps = {
+  routeProps: I.Map<string, any>,
```

Comment:
routeProps shouldn't leak to view components

Reply:
Sorry I'm still confused. This is used directly in `fs/routes.fs`, so I thought this would be a routed component. How do I differentiate a "view component" and a "routed component"?

Reply:
Ah I see. It's very rare for us to have plain components be directly routable. 99% of the time it's a connector. We can leave this but if it was me I'd made a hoc that does a mapProps and pass plain props to the view

Reply:
Got it; will do. Thanks!